frontend/app/src/assets/images/1inch.svg

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
![2020-12-28_22-33](https://user-images.githubusercontent.com/1658405/103244320-ebfd5880-495c-11eb-849c-c32cf5284e00.png)


you took their christmas SVG version. May need to change it later :stuck_out_tongue:

Reply:
Well it is was they have at their site :P

Reply:
Season to be jolly :)

Reply:
When Christmas rotki? :P